### PR TITLE
Added a table of contents for AAM series

### PR DESCRIPTION
**Pull request by fuadnafiz98**
I have added the table of contents for AAM series. 
Here is a screenshot of my changes
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +65,15 @@ Find out the videos here -
 
 ### Series 2 - AAM (An App a Month 🥭)
 
-Coming soon...
+Find out the videos here -
+
+| S.No   |      Topic      |  YouTube link                                                                          |
+|--------|-----------------|----------------------------------------------------------------------------------------|
+| 1      | Project 1       |  https://www.youtube.com/playlist?list=PLGyA74h_S9NqJvvQ7-l9bUHHS6bMCkgo0              |
```
Comment (MadhavBahl) [inline README.md]: Can you add the actual name of the project or playlist instead of Project 1, Project 2, etc?

**Review by MadhavBahl — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fuadnafiz98) [reply README.md]: Ok, I am working on it

Comment (fuadnafiz98) [reply README.md]: I've updated the changes.

File: README.md [hunk 2]
```diff
@@ -42,7 +42,7 @@
 I have created/will be creating the following courses/tutorials/learning series on my YouTube channel -
 
 1. Being-Better
-2. AAM (🥭) 
+2. AAM (🥭)
```
Comment (MadhavBahl) [inline README.md]:
```suggestion
2. AAM (🥭) 
```

No need for this, doesn't really matter
Rest LGTM,
Thanks for the contribution

**Comment by MadhavBahl** [comment]
Thanks @fuadnafiz98 
Merging this PR

**Pull request merged**